src/sentry_plugins/hipchat_ac/views.py

```diff
@@ -369,6 +397,7 @@ def new_f(request, *args, **kwargs):
         # put something here so that sentry does not overwrite it
         # with deny.
         resp['X-Frame-Options'] = 'allow'
+        resp['Content-Security-Policy'] = 'frame-ancestors https://sentry.hipchat.com/'
```

Comment:
We shouldn't need to also add this header here. And to be fair, for `getsentry`, this will not actually work and will be clobbered anyways. The above `X-Frame-Options` rule should be sufficient.

Also, this is a hardcoded value that would only work for us, so this pattern should be avoided and we'd want to leverage a configuration setting to define what this would be.